tests/snapshots/files__if_in_loop-optimize-sequential.snap

```diff
@@ -7,24 +7,27 @@ expression: visualization.result
   c1_: int = const 0;
   c2_: int = const 1;
   c3_: int = const 10;
-  v4_: int = id c1_;
-  v5_: int = id c2_;
-  v6_: int = id v0;
-  v7_: int = id c1_;
-  v8_: int = id c3_;
-.b9_:
-  v10_: bool = lt v6_ v5_;
-  v11_: int = select v10_ v5_ v7_;
+  v4_: bool = lt v0 c2_;
```

Comment:
Better inv code motion!

Reply:
Nice! The same improvement is also in the non-sequential version